armi/nuclearDataIO/cccc/isotxs.py

```diff
@@ -18,7 +18,7 @@
 ISOTXS is a binary file that contains multigroup microscopic cross sections.
 ISOTXS stands for  *Isotope Cross Sections*.
 
-ISOTXS files are often created by a lattice physics code such as MC2 or DRAGON and 
+ISOTXS files are often created by a lattice physics code such as MC2 or DRAGON and
```

Comment:
Let's watch how the docs build after your PR.  The white space sometimes matters in Sphinx RST docs.

After your PR merges, the API docs will go here:

https://terrapower.github.io/armi/.apidocs/modules.html